cmd/gdeploy/utils/linker.go

```diff
@@ -0,0 +1,14 @@
+package linker
+
+// Helper package to clean up the links to the docs in the gdeploy cli
+type Linker struct {
```

Comment:
We can remove this and use `url.URL` which is built in to the Go standard library for joining paths: 
```go
u := url.URL{
	Scheme: "https",
	Host: "docs.commonfate.io",
}
u.Path = "granted-approvals/sso/overview"
fmt.Printf("%s", u.String())
```

Separately - I usually request changes on using packages like `utils` as it's good to use a more specific package name in Go :) (but in this case we can remove the `utils` folder altogether)

Reply:
Perfect, didn't know about this functionality in `url.URL` thanks